src/database/config.js

```diff
@@ -0,0 +1,21 @@
+import pg from "pg";
+import dotenv from "dotenv";
+
+dotenv.config();
```

Comment:
Instead  of importing the entire dotenv library, you can use object destructuring to import config alone.

Something like
```import { config } from 'dotenv';```

```config();```

Reply:
That's cool